src/test/java/edu/cornell/kfs/concur/rest/services/ConcurAuthenticationServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package edu.cornell.kfs.concur.rest.services;
+
+import static org.junit.Assert.assertEquals;
+
+import org.apache.commons.lang.StringUtils;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.cornell.kfs.concur.service.impl.ConcurAuthenticationServiceImpl;
+
+public class ConcurAuthenticationServiceImplTest {
+
+    private static final String CONCUR_TOKEN = "concurToken";
+    private static final String BAD_TOKEN = "badToken";
+
+    private ConcurAuthenticationServiceImpl concurAuthenticationService;
+
+    @Before
+    public void setUp() {
+        concurAuthenticationService = new ConcurAuthenticationServiceImpl();
+        concurAuthenticationService.setConcurToken(CONCUR_TOKEN);
+    }
+
+    @Test
+    public void testIsConcurTokenValid_pass() {
```

Comment:
My preference would be to eliminate underscores from method names and just stick with camel casing throughout. Also, since the test is annotated, you could remote "test" from the start of the method name (that seems to be a newer test naming paradigm). So isConcurTokenValidPass or even isConcurTokenValidGoodToken / isConcurTokenValidBadToken / isConcurTokenValidEmptyToken for example.